src/bridge/mod.rs

```diff
@@ -185,22 +187,60 @@ impl<IS: Io> Bridge<IS> {
         }
 
         for ev in &sync.timeline.events {
```

Comment:
We'll also need to check in sync.state.events, in the case where the sync has been "limited"